### PR TITLE
Update Python dependencies

### PR DESCRIPTION
**Pull request by galaxybot**
Run `make update-dependencies`.

### CODE REVIEW

File: lib/galaxy/dependencies/pinned-requirements.txt [hunk 1]
```diff
@@ -86,7 +83,7 @@ humanfriendly==10.0 ; python_version >= "3.7" and python_version < "3.12"
 idna==3.4 ; python_version >= "3.7" and python_version < "3.12"
 importlib-metadata==4.13.0 ; python_version >= "3.7" and python_version < "3.12"
 importlib-resources==5.12.0 ; python_version >= "3.7" and python_version < "3.12"
-isa-rwval==0.10.10 ; python_version >= "3.7" and python_version < "3.12"
+isa-rwval==0.10.1 ; python_version >= "3.7" and python_version < "3.12"
```
Comment (nsoranzo) [inline lib/galaxy/dependencies/pinned-requirements.txt]: This downgrade is a bit suspicious.

**Comment by nsoranzo** [comment]
Ready to review/merge, the Integration Selenium failed test is a flaky one.

File: lib/galaxy/dependencies/pinned-requirements.txt [hunk 2]
```diff
@@ -1,19 +1,17 @@
---extra-index-url https://wheels.galaxyproject.org/simple
```
Comment (mvdbeek) [inline lib/galaxy/dependencies/pinned-requirements.txt:1]: Was that intentional ?

Comment (nsoranzo) [reply lib/galaxy/dependencies/pinned-requirements.txt:1]: Not sure what changed, but it was removed automatically when doing `poetry export ...` . I don't think it matters as we explicitly pass the `--extra-index-url` option to pip in `scripts/common_startup.sh` (and all tests still pass).

**Review by dannon — APPROVED** [review]

**Pull request merged**